cmd/ingester/main.go

```diff
@@ -76,6 +76,9 @@ func main() {
 			consumer.Start()
 
 			svc.RunAndThen(func() {
+				if err := options.TLS.Close(); err != nil {
```

Comment:
will this not be already included in storageFactory.Close()?

Reply:
The storage factory closes the TLS config for the producer. This closes the consumer.